tools/aequatus/aequatus.xml

```diff
@@ -0,0 +1,39 @@
+<tool id="aequatus-generator" name="aequatus parser" version="0.1.1">
```

Comment:
Yes good point I will change it to name="Aequatus" description=Generator. Because it Generates Aequatus datatype that we want to use in visualisation